File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.java
class Solution
{
    public int longestConsecutive(int[] nums)
    {
        Set<Integer> hashSet = new HashSet<>();
        List<Integer> l = new ArrayList<>();
        for (int num : nums)
        {
            hashSet.add(num);
        }
        int longest = 0;
        for (int num : hashSet)
        {
            if (! hashSet.contains(num - 1))
            {
                int len = 1;
                while (hashSet.contains(num + len))
                {
                    len++;
                }
                longest = Math.max(len, longest);
            }
        }

        return longest;
    }
}

File: Data Structures & Algorithms/maximum-ascending-subarray-sum/submission-2.cpp
class Solution {
public:
    int maxAscendingSum(vector<int>& nums) {
        int m = 0;
        int count = nums[0];
        for (int i = 1; i<nums.size(); i++)
        {
            if (nums[i-1] < nums[i])
            {
                count += nums[i];
            }
            else
            {
                m = max(m, count);
                count = nums[i];
            }
        }
                m = max(m, count);

        return m;
    }
};